lib/dialects/abstract/query-generator.js

```
@@ -1519,8 +1535,26 @@ const QueryGenerator = {
 
         mainQueryOrder.push(this.quote(t, model, false, '->'));
       }
+    } else if (typeof options.order === 'string') {
+      // split
+      for (let order of options.order.split(',')) {
+        // get options
+        const orderOption = getOrderOption(order);
+        if(orderOption.index > -1){
+          order = order.substring(0, orderOption.index);
+        }
+        order = _.trim(order);
+        let sql = this.quote(order, model, false, '->');
+        if(orderOption.option){
+          sql += ' ' + orderOption.option;
+        }
+        if (subQuery) {
+          subQueryOrder.push(sql);
+        }
+        mainQueryOrder.push(sql);
+      }
```

Wait, how is this removing the string order option? Passing a string to `order` should throw an exception. If you want to order by a literal, use `Sequelize.literal()`. Any custom sanitisation is prone to be spoofed.